site/content/docs/main/backup-hooks.md

```diff
@@ -78,14 +78,33 @@ velero backup get nginx-hook-test
 velero backup logs nginx-hook-test | grep hookCommand
 ```
 
-## Using Multiple Commands
+## Backup hook commands examples
+
+### Multiple commands
 
 To use multiple commands, wrap your target command in a shell and separate them with `;`, `&&`, or other shell conditional constructs.
 
 ```shell
     pre.hook.backup.velero.io/command='["/bin/bash", "-c", "echo hello > hello.txt && echo goodbye > goodbye.txt"]'
 ```
 
+#### Using environment variables
+
+You are able to use environment variables from your pods in your pre and post hook commands by including a shell command before using the environment variable. For example, `MYSQL_ROOT_PASSWORD` is an environment variable defined in pod called `mysql`. To use `MYSQL_ROOT_PASSWORD` in your pre-hook, you'd include a shell, like `/bin/sh`, before calling your environment variable:
```

Comment:
The shell is not necessary, the user can also run a compiled program that reads the environment variable in the code.  The snippet is still a valid example but we should just point out that the hook shares the environment variables that are set in the container.